src/dbally/audit/event_handlers/cli_event_handler.py

```diff
@@ -17,7 +17,21 @@
 
 class CLIEventHandler(EventHandler):
     """
-    CLI event handler interface.
+    This handler displays all interactions between LLM and user happending during `Collection.ask`\
+    execution inside the terminal.
+
+    ### Usage
+
+    ```python
+        import dbally
+        from dbally.audit.event_handlers.cli_event_handler import CLIEventHandler
+
+        dbally.use_event_handler(CLIEventHandler())
+    ```
+
+    from now, during every `Collection.ask` execution you will see output similar to the one below:
```

Comment:
"from now" -> "for now" -> "currently"

Reply:
I ment that after using CLIEventHandler by writing "from now". But I understand the amiguity